backend/layers/processing/logger.py

```diff
@@ -7,21 +7,25 @@
 
 from backend.common.logging_config import DATETIME_FORMAT, LOG_FORMAT
 
+logger = logging.getLogger("processing")
+
 
 def configure_logging(level=None):
     level = getattr(logging, os.environ.get("LOG_LEVEL", "INFO").upper())
     log_stdout_handler = logging.StreamHandler(stream=sys.stdout)
     formatter = jsonlogger.JsonFormatter(LOG_FORMAT, DATETIME_FORMAT)
     log_stdout_handler.setFormatter(formatter)
-    logging.basicConfig(handlers=[log_stdout_handler], level=level)
+    logging.basicConfig(handlers=[log_stdout_handler], level=level, force=True)
+    logging.getLogger("botocore").setLevel(max([logging.INFO, level]))  # don't set boto3 less than INFO
+    logging.getLogger("s3transfer").setLevel(max([logging.INFO, level]))
```

Comment:
doesn't this require a `s3transfer` logger exist already? Where is it? I'm not seeing it in the repo

Reply:
It does **not** need to be instantiated by the `s3transfer` library before being configured.

Reply:
Where is it being used? And how is it being accessed? I'm ignorant on this topic

Reply:
Ah I see. The `s3transfer` python package will actually pick up this logger to use for its own logging, right?

Reply:
exactly, and it will be the same one we configure on this line